lib/modules/notifications.js

```diff
@@ -58,7 +59,7 @@ addModule('notifications', function(module, moduleID) {
 					value: false
 				},
 				{
-					name: 'cooldown',
+					name: 'closeDelay',
```

Comment:
notificationType rows get mapped to objects using the field names, so instances of `.cooldown` would need to get updated if this were changed.

that said, `cooldown` is how many milliseconds before this notification type is allowed to be shown again